src/Form/ChoiceList/ModelChoiceLoader.php

```diff
@@ -40,70 +43,69 @@ final class ModelChoiceLoader implements ChoiceLoaderInterface
      */
     private $class;
 
+    /**
+     * @var string|null
+     */
     private $property;
 
+    /**
+     * @var mixed
+     */
     private $query;
```

Comment:
I would like to see this as `object|null` this will force people to use objects instead of arrays.

Reply:
Fixed